guides/source/action_controller_overview.md

```diff
@@ -394,10 +397,7 @@ Read more about session storage in the [Security Guide](security.html).
 If you need a different session storage mechanism, you can change it in an initializer:
 
 ```ruby
-# Use the database for sessions instead of the cookie-based default,
-# which shouldn't be used to store highly confidential information
-# (create the session table with "rails g active_record:session_migration")
-# Rails.application.config.session_store :active_record_store
+# Rails.application.config.session_store :cache_store
```

Comment:
Good change! :+1:  We might also consider linking to the configuration guide below this example.  Something like:

```
See the [configuration guide](configuring.html#config-session-store) for more information.
```